model/src/main/resources/reference.conf

```diff
@@ -9,6 +9,9 @@ ort {
 
   severeIssueThreshold = ERROR
 
+  enableRepositoryPackageCurations = true
```

Comment:
The commit message and PR message says this feature is disable by default but in the `reference.conf` you show it as enabled.  Should we set it to false and add a comment what these flags do.

Reply:
> The commit message and PR message says this feature is disable by default but in the `reference.conf` you show it as enabled. Should we set it to false and add a comment what these flags do.

The `reference.conf` does not reflect the default settings. See e.g. line 10 for `severeIssueThreshold`, which defaults to `WARNING`.

If the flag for `enableRepositoryPackageCurations` is not set at all, it is set to `false` and therefore disabled by default.
I did set it to `true` to check in a test if the changed value is taken into account.

> and add a comment what these flags do.

As of now the `reference.conf` does not document any of the other options.
I documented the feature in `docs/config-file-ort-yml.md`.